### DISCUSSION TITLE
App to read files on tablet/iPad

### DISCUSSION THREAD

**Discussion by lmgonzalezl** [musescore/MuseScore#22667] [Ideas]
### Your idea

To have an app that reads PDF, MuseScore, or XML files to use during concerts.

### Problem to be solved

Having just one application that provides musicians with everything they need on their tablet.
Right now, it's necessary to have MuseScore for composing and another app for reading the generated files, organizing song lists, etc.
I know this is a completely new application and not the purpose of MuseScore, but it would be interesting to have a sheet music reader similar to MusicReader that is connected to your MuseScore account.

**Comment by jeetee** [comment]
How do you envision this app to differ from the mobile viewer/player companion apps from the .com score sharing platform? ([Android](https://musescore.com/android) / [iOS](https://musescore.com/ios))

**Comment by lmgonzalezl** [comment]
It's just a personal opinion. Having a file reader that can connect to your MuseScore account and allows you to create playlists for viewing would add value; it would be everything a musician needs on their device (Android/iOS) for work, both in editing and playback, as well as for viewing during a concert.
As I said, it's just a simple personal opinion. It might be crazy what I'm suggesting, but craziness often leads to good ideas ;)

**Comment by jeetee** [comment]
But how would it differ from the already existing MuseScore mobile player apps as linked above? Those *do* link to your MuseScore account..

Unless you're actually asking about a mobile **editor** and not just a viewer.
